src/main/java/com/wellsky/workflow/WorkflowApplication.java

```diff
@@ -6,7 +6,14 @@
 @SpringBootApplication
 public class WorkflowApplication {
 
+    public String PASSWORD = "SECRETPASSWORD";
+
     public static void main(String[] args) {
+        //Bad variable name
+        String SuperHorrible_VarName="";
+        String bad="";
+        //System out should be a critical
+        System.out.println(SuperHorrible_VarName + bad);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS106)